datasets/ljspeech.py

```diff
@@ -0,0 +1,96 @@
+__all__ = [
+    "DownloadLJSpeechCorpusJob",
+]
```

Comment:
Does it make sense to just hase the Download Job here? Maybe also add the other job.